libs/feature/search/src/lib/sort-by/sort-by.component.ts

```diff
@@ -3,10 +3,12 @@ import { marker } from '@biesbjerg/ngx-translate-extract-marker'
 import { SortByEnum } from '@geonetwork-ui/util/shared'
 import { SearchFacade } from '../state/search.facade'
 import { SearchService } from '../utils/service/search.service'
+import { getMetadataQualityConfig } from '@geonetwork-ui/util/app-config'
```

Comment:
We try to avoid importing the app config from libs because components can exist without an app configuration (e.g. in web components); in this situation such an import might cause a crash.

See other comment by @fgravin